mcp_zammad/client.py

```
@@ -66,6 +69,20 @@ def __init__(
         )
+        if self.insecure:
+            # Allow connecting to instances with self-signed/missing CA certs.
+            session = getattr(self.api, "session", None)
+            if session is None:
+                connection = getattr(self.api, "_connection", None)
+                session = getattr(connection, "session", None) if connection is not None else None
+            if session is None:
+                raise ConfigException(
+                    "ZAMMAD_INSECURE is enabled but the installed zammad-py client does not expose a "
+                    "requests session; TLS verification cannot be disabled."
+                )
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)